Lib/test/test_winconsoleio.py

```diff
@@ -92,9 +92,11 @@ def test_open_name(self):
         f.close()
         f.close()
 
-        f = open('C:/con', 'rb', buffering=0)
-        self.assertIsInstance(f, ConIO)
-        f.close()
+        # Windows 11 changed MS-DOS device name handling
+        if sys.getwindowsversion()[:3] < (10, 0, 22000):
+            f = open('C:/con', 'rb', buffering=0)
+            self.assertIsInstance(f, ConIO)
+            f.close()
```

Comment:
Can you please modify the test to use "with f: ..."?

Reply:
If you notice, the rest of the test cases do not do that.  Changing to context managers should probably be done as another issue.  The only change here is moving the existing test into the if-block.

Reply:
I see. Leave it as it is.